website/docs/docs/collaborate/explore-projects.md

```diff
@@ -161,34 +187,41 @@ An example of the details you might get for an exposure:
 - **General** tab includes:
-    - **Lineage** graph &mdash; The test’s lineage graph that you can interact with. The graph includes one parent node and one child node from the test resource. Click the Expand icon in the graph's upper right corner to view the test in full lineage graph mode.
+    - **Lineage** graph &mdash; The test’s lineage graph that you can interact with. The graph includes one upstream node and one downstream node from the test resource. Click the Expand icon in the graph's upper right corner to view the test in full lineage graph mode.
     - **Description** section &mdash; A description of the test.
     - **Recent** section &mdash; Information on the last time the test ran, how long it ran for, whether the test passed, the job ID, and the run ID.
     - **Details** section &mdash; Details like schema, severity, package, and more.
     - **Relationships** section &mdash; The nodes the test **Depends On**.
 - **Code** tab &mdash; The source code and compiled code for the test.
 
+</expandable>
 
-### Example of source details
-
-An example of the details you might get for each source table within a source collection:
+<expandable alt_header="What details are available for each source table within a source collection?">
 
 - Status bar (below the page title) &mdash; Information on the last time the source was updated and the number of tables the source uses. 
 - **General** tab includes:
-    - **Lineage** graph &mdash; The source’s lineage graph that you can interact with. The graph includes one parent node and one child node from the source. Click the Expand icon in the graph's upper right corner to view the source in full lineage graph mode.
+    - **Lineage** graph &mdash; The source’s lineage graph that you can interact with. The graph includes one upstream node and one downstream node from the source. Click the Expand icon in the graph's upper right corner to view the source in full lineage graph mode.
     - **Description** section &mdash; A description of the source.
     - **Source freshness** section &mdash; Information on whether refreshing the data was successful, the last time the source was loaded, the timestamp of when a run generated data, and the run ID.
     - **Details** section &mdash; Details like database, schema, and more.
     - **Relationships** section &mdash; A table that lists all the sources used with their freshness status, the timestamp of when freshness was last checked, and the timestamp of when the source was last loaded.
 - **Columns** tab &mdash; The available columns in the source. This tab also shows tests results (if any) that you can select to view the test's details page. A :white_check_mark: denotes a passing test.
 
+</expandable>
+
+### Example of model details
+
+Example of the details view for the model `supplies`: 
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-model-details.png" width="100%" title="Example of resource details" />
```

Comment:
this screenshot is hard to see, even when blown up. i leaned towards capturing all the details on one page (which is why it's hard to see) but maybe that was the wrong call. should i recapture?